CondFormats/JetMETObjects/bin/JetResolution_t.cc

```diff
@@ -61,8 +61,9 @@ int main(int argc, char** argv) {
   cout << "nevts:    " << nevts << endl;
   cout << "gaussian: " << doGaussian << endl << endl;
 
-  string cmssw_base(std::getenv("CMSSW_BASE"));
-  string cmssw_release_base(std::getenv("CMSSW_RELEASE_BASE"));
+  // CMSSW_BASE and CMSSW_RELEASE_BASE are always set
+  [[clang::suppress]] string cmssw_base(std::getenv("CMSSW_BASE"));
+  [[clang::suppress]] string cmssw_release_base(std::getenv("CMSSW_RELEASE_BASE"));
```

Comment:
@iarspider , we can just use FileInPath here and simplify the code here e.g see https://github.com/cms-sw/cmssw/pull/46258/files#diff-91dfa2ade2750e8b3fe5f8668547b42beb3c49d344c897011e0c619e0a149d7a as ref